app/models/preserved_copy.rb

```diff
@@ -3,13 +3,19 @@
 class PreservedCopy < ApplicationRecord
   belongs_to :preserved_object
   belongs_to :endpoint
-  belongs_to :status
 
   validates :preserved_object, presence: true
   validates :endpoint, presence: true
   validates :version, presence: true
-  validates :status, presence: true
   # NOTE: size here is approximate and not used for fixity checking
   validates :size, numericality: { only_integer: true, greater_than: 0 }, allow_nil: true
-
+  enum status: {
+    ok: 0,
+    invalid_moab: 1,
+    invalid_checksum: 2,
+    not_found_on_disk: 3,
+    expected_version_not_found_on_disk: 4,
+    fixity_check_failed: 5
+  }
+  validates :status, inclusion: { in: statuses.keys }
```

Comment:
I am a bit uncomfy using plain strings in the code.  Could we turn those strings into, say:  STATUS_OK here?

will something like this fly?
```
STATUS_OK = status.ok
```

Then we could have 
```
STATUS_DEFAULT = STATUS_OK
```